src/components/ModalPayment/ModalCardDetails/ModalCardDetails.tsx

```diff
@@ -203,16 +224,39 @@ const ModalCardDetails = ({
           'modalPayment.modalCardDetails.disclaimer.gift_card',
           sellerName
         );
+      case ModalPaymentTypes.modalPages.light_up_chinatown:
+        return t('modalPayment.modalCardDetails.disclaimer.light_up_chinatown');
       default:
         break;
     }
   };
 
+  const setDetailsText = (
+    type: string | ModalPaymentTypes.modalPages,
+    amount: number
+  ) => {
+    if (sellerId === 'send-chinatown-love')
+      type = ModalPaymentTypes.modalPages.donation_pool;
+
+    let message;
+
+    if (
+      type === ModalPaymentTypes.modalPages.gift_card ||
+      (ModalPaymentTypes.modalPages.light_up_chinatown && amount >= 45)
```

Comment:
I think you forgot: 
```suggestion
      (type === ModalPaymentTypes.modalPages.light_up_chinatown && amount >= 45)
```

Reply:
You can also get rid of the temporary `message` variable and return directly
```suggestion
         if (
      type === ModalPaymentTypes.modalPages.gift_card ||
      (type === ModalPaymentTypes.modalPages.light_up_chinatown && amount >= 45)
    ) {
      return t('modalPayment.modalCardDetails.details.voucher');
    } 
    return t('modalPayment.modalCardDetails.details.donation');
```